### DISCUSSION TITLE
WezTerm Applescript Compatability

### DISCUSSION THREAD

**Discussion by aesbetic** [wezterm/wezterm#4135] [Q&A]
I was looking for a way to make neovim in wezterm my default editor on mac os. Has anyone achieved this? If so, i would love to know how. Wezterm doesn't have a dictionary in applescript and i'm not all that proficient with applescript so i'm kinda stuck.

**Comment by karimlevallois** [comment]
> I was looking for a way to make neovim in wezterm my default editor on mac os. Has anyone achieved this? If so, i would love to know how. Wezterm doesn't have a dictionary in applescript and i'm not all that proficient with applescript so i'm kinda stuck.

I’ve set up a shell script to open files inside Neovim which seems to work OK, but I would much prefer to find a way of using Apple Script to open the file in Neovim and then activate full screen and change the font size to 14.
